core/templates/dev/head/AppInit.ts

```diff
@@ -16,14 +16,31 @@
  * @fileoverview File for initializing the main oppia module.
  */
 
-import { NgModule } from '@angular/core';
+import 'core-js/es7/reflect';
+import 'zone.js';
+
+import { Component, NgModule, StaticProvider } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-import { StaticProvider } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+// This component is needed to force bootstrap Angular at the beginning of the
```

Comment:
"force bootstrap" --> "force-bootstrap"

(if that's what you mean)

Reply:
Done!